react/src/components/Note.jsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 
-function Note() {
+function Note(props) {
```

Comment:
More properties should be passed here (e.x: `backgoundColor`, `created_at`). But they can be added as we build the `Post` functionality.

Reply:
Yes, we can add them in future tickets. I think it is fine for now.